MID Server/MID Server Script Includes/IllumioMIDConstants.js

```diff
@@ -12,6 +12,7 @@ var METHOD_CLASSES = {
     DELETE: DeleteMethod
 };
 var SUCCESS_CODES = [200, 201, 202, 204];
```

Comment:
Don't use a hardcoded list like this for successful responses, just check that the code is in the range 200-299. The PCE may implement a 2xx code that isn't in this list later, and we don't want to have to come and update this list every time.